packages/react/src/hooks/utils.ts

```diff
@@ -16,6 +16,7 @@ const clerkLoaded = (isomorphicClerk: IsomorphicClerk) => {
  * @internal
  */
 export const createGetToken = (isomorphicClerk: IsomorphicClerk) => {
+  // eslint-disable-next-line
```

Comment:
❓ Mind explicitly defining what we're disabling here? Thanks!

Reply:
Before it was giving me an eslint error for using `explicit any`, now that's no longer the case (don't know why tho), so i'm removing this line completely